third-party/dnd-wiki/classes/class-weapon-master.xml

```diff
@@ -0,0 +1,452 @@
+			<grant type="Class Feature" id="ID_BREW_CLASS_FEATURE_WEAPON_MASTER_IMPOSING_FIGHTER" level="14" />
+		</rules>
+		<multiclass id="ID_BREW_MULTICLASS_WEAPON_MASTER">
+			<prerequisite>Strength 15 or Dexterity 15, and Constitution 13</prerequisite>
+			<requirements>([str:15]||[dex:15]),[con:13]</requirements>
+			<setters>
+				<set name="multiclass proficiencies">Light armor, medium armor, simple weapons, and 1 martial weapon</set>
+			</setters>
+			<rules>
+				<grant type="Grants" id="ID_INTERNAL_GRANT_MULTICLASS" />
+				<grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_LIGHT_ARMOR" />
+				<grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_MEDIUM_ARMOR" />
+				<grant type="Proficiency" name="ID_PROFICIENCY_WEAPON_PROFICIENCY_SIMPLE_WEAPONS" />
+				<select type="Proficiency" name="Martial Weapon Proficiency (Weapon Master)" supports="ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_MELEE||ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_RANGED" number="1"/>
+			</rules>
+		</multiclass>
+	</element>
+	<element name="Favored Weapon" type="Class Feature" source="D&amp;D Wiki" id="ID_BREW_CLASS_FEATURE_WEAPON_MASTER_FAVORED_WEAPON">
+		<description>
+			<p>For a Weapon master, your weapons are what makes you who you are. Through your life you have adapted to fight with all kinds of weapons but have specific type of weapon as your signature weapons.</p>
+			<p class="indent">You get +1 bonus to attack and damage rolls when you use your favored weapon. Choose one simple or martial weapon type, and another when you reach 11th level.</p>
+			<p class="indent">When you reach 20th level, you have advantage on attacks with your favored weapon.</p>
+		</description>
+		<sheet>
+			<description>You get +1 bonus to attack and damage rolls when you use your favored weapon.</description>
+			<description level="11">You get +1 bonus to attack and damage rolls with two types of weapons as your favored weapons.</description>
+			<description level="20">You get +1 bonus to damage and attack rolls with your favored weapons, you also have advantage on attacks with them.</description>
+		</sheet>
+	</element>
+	<element name="Keen Eye" type="Class Feature" source="D&amp;D Wiki" id="ID_BREW_CLASS_FEATURE_WEAPON_MASTER_KEEN_EYE">
+		<description>
+			<p>At 2nd level, you gain advantage on Investigation checks related to weapons, armor, other related metal pieces.</p>
+		</description>
+		<sheet>
+			<description>You gain advantage on Investigation checks related to weapons, armor, other related metal pieces.</description>
+		</sheet>
+	</element>
+	<element name="Fighting Style" type="Class Feature" source="D&amp;D Wiki" id="ID_BREW_CLASS_FEATURE_WEAPON_MASTER_FIGHTING_STYLE">
+		<description>
+			<p>At 2nd level, you adopt a style of fighting as your specialty. Choose one of the following options. You canвЂ™t take a Fighting Style option more than once, even if you later get to choose again.</p>
```

Comment:
"canвЂ™t", is my problem with code, where "**'**" will show up as "?" in app, if i don't change it to "**вЂ™**", i don't know why. Maybe it's just my old Windows again.
I agree about fighting styles, it was like that when i was copying from Blood Hunter class (if it's not obvious enough), but later i did change rule line to follow all fighting styles and not just these. Should i just remove those lines all together?

Reply:
I saw it as exactly "canвЂ™t", so potentially get a third opinion. (How did you discover that was the character combination?)

Potentially alter the text so it says to pick from the Fighter/Ranger fighting styles.

Reply:
I already discussed it with swdriessen, i don't know if this because of my language settings, Windows, or maybe because i was using WordPad to edit. But yeah... it's "canвЂ™t" in code, but "can't" in Aurora. Strangely, i think GitHub Desktop corrected those in Lore Mastery pull.

I don't know about altering them. Now i think, this is basic description for any classes, even if they include other "non-Core Books" Fighting Styles.

Reply:
Okay, i fixed `вЂ™`. There shouldn't be anything like that anymore.